crates/kitsune_p2p/kitsune_p2p/src/spawn/actor.rs

```diff
@@ -210,24 +209,15 @@ impl KitsuneP2pActor {
 
 async fn create_meta_net(
     config: &KitsuneP2pConfig,
-    tls_config: tls::TlsConfig,
+    _tls_config: tls::TlsConfig,
     internal_sender: ghost_actor::GhostSender<Internal>,
     host: HostApiLegacy,
-    metrics: Tx2ApiMetrics,
+    _metrics: Tx2ApiMetrics,
```

Comment:
Can these parameters not be removed if they are no longer used?

Haven't dug into it but if it's a Tx2 specific metrics thing, can that not be removed too or is there something we want to bring forward and integrate with opentelemetry now with some metrics set up for tx5?

Reply:
I know this PR claims to remove *all* vestiges of tx2, but I also wanted to not cut in too deeply. @neonphog can you weigh in on this one?

Reply:
I'm always a fan of tackling things iteratively. Remove some tx2 stuff is better than never getting the PR merged : )